frame/dapps-staking/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pallet-dapps-staking"
-version = "2.0.1"
```

Comment:
You've added a new storage item so it's not a minor change anymore.
I think you can bump it up to 2.1.0.